piicatcher/scanner.py

```diff
@@ -51,18 +53,23 @@ class ColumnNameRegexDetector(MetadataDetector):
         Email: re.compile("^.*(email|e-mail|mail).*$", re.IGNORECASE),
         BirthDate: re.compile(
             "^.*(date_of_birth|dateofbirth|dob|"
-            "birthday|date_of_death|dateofdeath).*$",
+            "birthday|date_of_death|dateofdeath|birthdate).*$",
             re.IGNORECASE,
         ),
         Gender: re.compile("^.*(gender).*$", re.IGNORECASE),
         Nationality: re.compile("^.*(nationality).*$", re.IGNORECASE),
         Address: re.compile(
-            "^.*(address|city|state|county|country|zipcode|postal|zone|borough).*$",
+            "^.*(address|city|state|county|country|zone|borough).*$",
+            re.IGNORECASE,
+        ),
+        ZipCode: re.compile(
+            "^.*(zipcode|zip_code|postal|postal_code|zip).*$",
             re.IGNORECASE,
         ),
         UserName: re.compile("^.*user(id|name|).*$", re.IGNORECASE),
         Password: re.compile("^.*pass.*$", re.IGNORECASE),
-        SSN: re.compile("^.*(ssn|social).*$", re.IGNORECASE),
+        SSN: re.compile("^.*(ssn|social_number).*$", re.IGNORECASE),
```

Comment:
```suggestion
        SSN: re.compile("^.*(ssn|social_number|social_security).*$", re.IGNORECASE),
```

To cover column names like "social_security_number" or "social_security_no".